pkg/auth/auth_test.go

```diff
@@ -91,7 +91,7 @@ func TestMakeXRegistryConfigHeaderGetCredentialsRoundtrip(t *testing.T) {
 		require.NoError(t, err, tc.name)
 		defer RemoveAuthfile(resPath)
 		if tc.expectedOverride == nil {
-			assert.Nil(t, override, tc.name)
+			assert.NoError(t, override, tc.name)
```

Comment:
This is not an error

Reply:
I guess globally find . --exec sed ... was not a great idea, will fix.